app/pages/SearchResultsPage/SearchResultsPage.tsx

```diff
@@ -146,6 +146,10 @@ const InnerSearchResults = ({
   setSearchRadius: (radius: string) => void;
   untranslatedQuery: string | undefined | null;
 }) => {
+  const [aroundLatLng, setAroundLatLng] = useState(
+    userLocation ? `${userLocation.lat}, ${userLocation.lng}` : null
+  );
```

Comment:
Nit/minor comment: It's a little weird to use a string with a comma separating the two values as the internal state representation, since it kind of imposes a requirement on both the setter and getter sides to turn the more natural representation of having two separate numbers into a string and vice versa. From reading the code, I realize that the Algolia InstantSearch components wants the string representation, but maybe we should store it in our state as two separate numbers?

Maybe this could just be an object with the following TypeScript shape?

```ts
type Location = { lat: number; lng: number };
```

Then, the only place where we would convert it to a string is at the `<Configure aroundLatLng={...} ... />` instantiation site, and everywhere else we just pass in the two numbers?

This isn't super critical, but just a thought on how to represent this a bit more cleanly.

Reply:
Yes! I appreciate the feedback. That sounds logical. I've updated it.